samples/app/src/main/java/com/sourcepoint/app/v6/di/AppModule.kt

```diff
@@ -39,7 +39,8 @@ val appModule = module {
             customVendorList = customVendorDataList,
             customCategories = customCategoriesData,
             useGdprGroupPmIfAvailable = false,
-            authId = null //get(qualifier = named("authId")))
+            authId = null, //get(qualifier = named("authId")))
+            diagnostic = emptyList()
```

Comment:
can we leave comma at the end of the constructor param list, just so next time we want to add a new param to the constructor, it will just appear as an addition of one line, and not changing one and adding one?